conans/client/graph/proxy.py

```diff
@@ -113,6 +113,11 @@ def _download_recipe(self, layout, ref, output, remotes, remote, recorder):
         def _retrieve_from_remote(the_remote):
             output.info("Trying with '%s'..." % the_remote.name)
             # If incomplete, resolve the latest in server
+            if "https://conan.bintray.com" in the_remote.url:
+                output.warn("Remote https://conan.bintray.com is deprecated and will be shut down "
+                            "soon.")
+                output.warn("Please use the new 'conancenter' default remote "
+                           "(https://center.conan.io).")
```

Comment:
For these messages, I really find it useful to provide a copy/paste solution to users:

```suggestion
                output.warn("Please use the new 'conancenter' default remote. Add it to your remotes with "
                           "'conan remote add --insert 0 conancenter https://center.conan.io'.")
```